packages/utils/getPackageRequestPeriods.js

```diff
@@ -0,0 +1,42 @@
+const { format: formatDate, subDays } = require('date-fns');
+
+const maxRequestPeriod = 365; // ~1 year
+
+function maxDate(a, b) {
+  return new Date(Math.max(a.getTime(), b.getTime()));
+}
+
+/**
+ * @param startDay {number} Start of period, 1 is yesterday
+ * @param endDay   {number} End of period 0 is today
+ * @returns {Array<{startDate: string, endDate: string}>}
+ */
+function getPackageRequestPeriods(startDay, endDay) {

```

Comment:
This is the part that I'm not too sure, I hope I mimic the old behavior. I added some tests, but I'm not really sure if I covered all the cases

Reply:
🙏 looks fine!